<?xml version="1.0"?>
<launch>

  <!-- start left camera -->
  <node pkg="cob_camera_sensors" type="color_camera" name="color_camera" ns="camera/left" cwd="node" respawn="false" output="screen">
	<param name="configuration_files" value="../../common/files/cob3-1/"/>
	<param name="camera_index" value="1"/>
  </node>

  <!-- start poller for left camera -->
  <node pkg="polled_camera" type="poller" name="poller" ns="camera/left" respawn="false" output="screen" args="3 camera:= output:=color_camera_stream"/>

</launch>
